### PR TITLE
Update comments with better laid out examples

### PR DESCRIPTION
**Pull request by iHiD**
These are all auto-approve comments. I've expanded them to use examples spread over multiple lines and included a link as suggested by @F3PiX.

### CODE REVIEW

**Comment by kotp** [comment]
I don't see an entry where they use variable interpolation (also known as string interpolation) and it is suggested to explore the `format`/`sprintf`/`String#%` options and benefits.

File: automated-comments/ruby/two-fer/kernel_format.md [hunk 1]
```diff
@@ -1 +1,9 @@
-As well as the `format` method, a common way to create strings in Ruby is to use interpolation. For example, rather than `format("One for %%s, one for me.", %<name_variable>s)` you might consider `"One for #{%<name_variable>s}, one for me."`.
+As well as the `format` method, a common way to create strings in Ruby is to use [string interpolation](http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html). For example:
+
+```ruby
```
Comment (ErikSchierboom) [inline automated-comments/ruby/two-fer/kernel_format.md]: Isn't giving the solution going against the suggested automated-mentoring guidelines: https://github.com/exercism/automated-mentoring-support/pull/61/files#diff-c3fc7e01c1c858a88579c56982ac5ba5R60?

File: automated-comments/ruby/two-fer/string_building.md [hunk 2]
```diff
@@ -1 +1,9 @@
-In Ruby, it is unusual to use string addition. People tend to use `sprintf`, `format`, `%%`, or string interpolation. So instead of `"One for " + %<name_variable>s + ", one for me."`, consider `"One for #{%<name_variable>s}, one for me."`.
+String concatenation is perfectly valid within Ruby, but a more common way to compose a string is with [string interpolation](http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html). For example:
+
+```ruby
```
Comment (ErikSchierboom) [inline automated-comments/ruby/two-fer/string_building.md]: Isn't giving the solution going against the suggested automated-mentoring guidelines: https://github.com/exercism/automated-mentoring-support/pull/61/files#diff-c3fc7e01c1c858a88579c56982ac5ba5R60?

File: automated-comments/ruby/two-fer/string_format.md [hunk 3]
```diff
@@ -1 +1,9 @@
-As well as the `String#%%` method, a common way to create strings in Ruby is to use interpolation. For example, rather than `"One for %%s, one for me." %% %<name_variable>s` you might consider `"One for #{%<name_variable>s}, one for me."`.
+As well as the `String#%%` method, a common way to create strings in Ruby is to use [string interpolation](http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html). For example:
+
+```ruby
```
Comment (ErikSchierboom) [inline automated-comments/ruby/two-fer/string_format.md]: Isn't giving the solution going against the suggested automated-mentoring guidelines: https://github.com/exercism/automated-mentoring-support/pull/61/files#diff-c3fc7e01c1c858a88579c56982ac5ba5R60?

File: automated-comments/ruby/two-fer/string_building.md [hunk 4]
```diff
@@ -1 +1,9 @@
-In Ruby, it is unusual to use string addition. People tend to use `sprintf`, `format`, `%%`, or string interpolation. So instead of `"One for " + %<name_variable>s + ", one for me."`, consider `"One for #{%<name_variable>s}, one for me."`.
+String concatenation is perfectly valid within Ruby, but a more common way to compose a string is with [string interpolation](http://ruby-for-beginners.rubymonstas.org/bonus/string_interpolation.html). For example:
```
Comment (ErikSchierboom) [inline automated-comments/ruby/two-fer/string_building.md]: A small note: this comment is named `string_building`, but aren't the other options also about string building? Maybe `string_concatenation` would be a bit more precise?

**Comment by iHiD** [comment]
> Isn't giving the solution going against the suggested automated-mentoring guidelines: 

@ErikSchierboom For comments that come with approval, I argue there is no "solution" to give. We're simply suggesting an improvement that the person can learn from, rather than giving the person a next step they need to take. I don't want to give someone a blog post to read for a trivial thing that takes 2secs to work out if you see an example.

In the notes it says "However, if the discovery **doesn't** trigger a dopamine hit, it makes total sense to show what it looks like.". That's from [this comment](https://github.com/exercism/automated-mentoring-support/issues/59#issuecomment-506728305)) and came specifically in relation to this example.

**Comment by iHiD** [comment]
> I don't see an entry where they use variable interpolation (also known as string interpolation) and it is suggested to explore the format/sprintf/String#% options and benefits.

@kotp Yeah - the analyzer doesn't currently do this.

File: automated-comments/ruby/two-fer/string_building.md (hunk 4 above)
Comment (iHiD) [reply automated-comments/ruby/two-fer/string_building.md]: Nice. Yes. Done, thanks.

File: automated-comments/ruby/two-fer/string_format.md (hunk 3 above)
Comment (SleeplessByte) [reply automated-comments/ruby/two-fer/string_format.md]: > However, if the discovery **doesn't** trigger a dopamine hit, it makes total sense to show what it looks like.

I think that's the case here.

**Pull request merged**